### PR TITLE
Updated reCAPTCHA with PHP config note

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) added a note regarding required PHP setting for Google reCAPTCHA. This update was vetted and approved by Rick Tempesta @phoenix128 

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/security/google-recaptcha.html

### CODE REVIEW

File: guides/v2.3/security/google-recaptcha.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ The Google reCAPTCHA extension is bundled and installed with {{site.data.var.ce}
 
 ## Configure reCAPTCHA
 
+{:.bs-callout .bs-callout-warning}
+Before reCAPTCHA can be configured, you must ensure that your `PHP.ini` file includes the following setting: `allow_url_fopen = 1`. See [Required PHP Settings]({{page.baseurl}}/install-gde/prereq/php-settings.html}}).
```
Comment (shrielenee) [inline guides/v2.3/security/google-recaptcha.md]:
```suggestion
Before reCAPTCHA can be configured, ensure that your `PHP.ini` file includes this setting: `allow_url_fopen = 1`. See [Required PHP Settings]({{page.baseurl}}/install-gde/prereq/php-settings.html}}).
```

**Comment by hickskenh** [comment]
running tests

**Comment by hickskenh** [comment]
running tests

**Comment by hickskenh** [comment]
running tests

**Pull request merged**